src/main/java/step3/domain/LottoNumber.java

```diff
@@ -1,28 +1,41 @@
 package step3.domain;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Optional;
 
 public class LottoNumber {
 
-    private List<Integer> lottoNumber;
+    public static final ArrayList<Integer> RANGE = new ArrayList(Arrays.asList(1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
+                                                                            11, 12, 13, 14, 15, 16, 17, 18, 19, 20,
+                                                                            21, 22, 23, 24, 25, 26, 27, 28, 29, 30,
+                                                                            31, 32, 33, 34, 35, 36, 37, 38, 39, 40,
+                                                                            41, 42, 43, 44, 45));
 
-    public LottoNumber(List<Integer> lottoNumber) {
-        this.lottoNumber = new ArrayList<>(lottoNumber);
+    private final int lottoNumber;
+
+    public LottoNumber(int lottoNumber) {
+        if(!isValid(lottoNumber)){
+            throw new IllegalArgumentException("로또 숫자 범위가 아닙니다.");
+        }
+        this.lottoNumber = lottoNumber;
     }
 
-    public List<Integer> getLottoNumber() {
+    public int getLottoNumber() {
         return lottoNumber;
     }
 
-    public boolean hasBonusNumber(int bonusNumber) {
-        return lottoNumber.contains(bonusNumber);
+    private boolean isValid(int lottoNumber){
+        return RANGE.contains(lottoNumber);
     }
 
     @Override
     public String toString() {
-        return lottoNumber.toString();
+        return lottoNumber+"";
```

Comment:
```suggestion
        return String.valueOf(lottoNumber);
```
차이는 없는데 개인적으론 위 방법이 조금 더 깔끔하다고 생각해요.
윤님이 적당히 선택해주시면 됩니다.